app/pages/clinicworkspace/ClinicPatients.js

```diff
@@ -2525,50 +2527,52 @@ export const ClinicPatients = (props) => {
   const renderTimeInRangeDialog = useCallback(() => {
     const timeInRangeFilterOptions = [
       {
+        title: t('Very Low'),
         value: 'timeInVeryLowPercent',
         threshold: glycemicTargetThresholds.timeInVeryLowPercent.value,
         prefix: t('Greater than'),
-        tag: t('Severe hypoglycemia'),
         rangeName: 'veryLow',
       },
       {
-        value: 'timeInLowPercent',
-        threshold: glycemicTargetThresholds.timeInLowPercent.value,
+        title: t('Low'),
+        value: 'timeInAnyLowPercent',
+        threshold: glycemicTargetThresholds.timeInAnyLowPercent.value,
         prefix: t('Greater than'),
-        tag: t('Hypoglycemia'),
-        rangeName: 'low',
+        rangeName: 'anyLow',
       },
       {
+        title: t('Not meeting TIR'),
         value: 'timeInTargetPercent',
         threshold: glycemicTargetThresholds.timeInTargetPercent.value,
         prefix: t('Less than'),
-        tag: t('Normal'),
         rangeName: 'target',
       },
       {
-        value: 'timeInHighPercent',
-        threshold: glycemicTargetThresholds.timeInHighPercent.value,
+        title: t('High'),
+        value: 'timeInAnyHighPercent',
+        threshold: glycemicTargetThresholds.timeInAnyHighPercent.value,
         prefix: t('Greater than'),
-        tag: t('Hyperglycemia'),
-        rangeName: 'high',
+        rangeName: 'anyHigh',
       },
       {
+        title: t('Very High'),
         value: 'timeInVeryHighPercent',
         threshold: glycemicTargetThresholds.timeInVeryHighPercent.value,
         prefix: t('Greater than'),
-        tag: t('Severe hyperglycemia'),
         rangeName: 'veryHigh',
       },
     ];
 
     if (showExtremeHigh) timeInRangeFilterOptions.push({
+      title: t('Highest'),
       value: 'timeInExtremeHighPercent',
       threshold: glycemicTargetThresholds.timeInExtremeHighPercent.value,
       prefix: t('Greater than'),
-      tag: t('Extreme hyperglycemia'),
       rangeName: 'extremeHigh',
     });
 
+    timeInRangeFilterOptions.reverse();
```

Comment:
Tiny nit: could just define the array in the new intended order rather than keeping the original, and then immediately reversing it.